src/components/home/partners/PrimaryPartner.jsx

```diff
@@ -1,8 +1,83 @@
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+
+const sponsorValues = [
+  {
+    title: 'Advocacy',
+    description:
+      "We amplify your voice and bridge industry and government to drive policies that ensure the growth and success of the region's tech industry."
+  },
+  {
+    title: 'Community',
+    description:
+      'We bring the regional tech community together through in-person and online events to deepen relationships and build meaningful connections.'
+  },
+  {
+    title: 'Visibility',
+    description:
+      'We shine a spotlight on the companies and people that power the region’s innovation economy and leverage our platform to promote the local tech sector.'
+  },
+  {
+    title: 'Resources',
+    description:
+      'We connect our members to resources to help their businesses grow and succeed through public and private sector partnerships.'
+  }
+];
+
+const displaySponsorValuesInGrid = (value, index) => {
+  return (
+    <Grid value xs={6} key={index}>
```

Comment:
Yeah, it's throwing this error for me:
```
PrimaryPartner.jsx:31 
Warning: Failed prop type: The prop `xs` of `Grid` can only be used together with the `item` prop.
```

Also, could we change this to 12 instead of 6, so it fits small viewports better? This line would then be:
```
    <Grid item value xs={12} sm={6} key={value.title}>
```

Reply:
changed in next push. I like that mobile display more, good call.